src/web/modules/services/detail.njk

```diff
@@ -110,6 +114,12 @@
       href: "/services/" + serviceId + "/link_accounts"
       })
     }}
+    {{ govukButton({
+      text: "Toggle redirect to service on terminal state flag",
+      classes: "govuk-button--warning",
```

Comment:
maybe we should make it clear why this is such a scary thing to do

Reply:
@mrlumbu good suggestion! https://github.com/alphagov/pay-toolbox/pull/165/commits/5dea91d8534072b06770da71783d975135ffe374